services/src/main/scala/cromwell/pipeline/service/ProjectService.scala

```diff
@@ -34,11 +34,36 @@ class ProjectService(projectRepository: ProjectRepository)(implicit executionCon
     projectRepository.addProject(project)
   }
 
-  def deactivateProjectById(projectId: ProjectId): Future[Option[Project]] =
-    for {
-      _ <- projectRepository.deactivateProjectById(projectId)
-      getProject <- projectRepository.getProjectById(projectId)
-    } yield getProject
+  def deactivateProjectById(projectId: ProjectId, userId: UserId): Future[Option[Project]] = {
+    val result = projectRepository.getProjectById(projectId)
```

Comment:
1) You already have a separate method `getProjectById` in this `Service`. Why don't you use it?
2) Why do we need a variable with such a meaningful name? :)

Reply:
fixed

Reply:
Nothing has changed. Therefore, nothing is fixed.

Reply:
Sorry, I didn't push. Fixed now

Reply:
Sorry, but I don't see any changes. Variable with a meaningless name still exists and you are still using the method  `projectRepository.getProjectById` directly.

Reply:
fixed